packages/schemas/index.js

```diff
@@ -55,7 +55,7 @@ const createModels = async () => {
   const model = await manager.toPublished();
 
   // Write published model to JSON file
-  await writeFile("./lib/model.json", JSON.stringify(model));
+  await writeFile("../dapp/model.json", JSON.stringify(model));
```

Comment:
It would be better if you keep "./lib/model.json" and then in your dapp or your server you can import the model like this:
`import publishedModel from "@scaffold-eth/schemas"`

Reply:
Done!